### ISSUE TITLE
Can esrgan use the 4x-ultrasharp model?

### ISSUE THREAD

**Issue by Damarcreative** [xinntao/ESRGAN#139]

**Comment by crapthings** [comment]
yes how to make this work?

<img width="1035" alt="image" src="https://github.com/xinntao/ESRGAN/assets/1147704/bcd76c78-4289-44cb-966b-5e0f68132fad">

<img width="1388" alt="image" src="https://github.com/xinntao/ESRGAN/assets/1147704/24a3151f-e544-4226-9001-a6b694a034fe">

**Comment by Damarcreative** [comment]
@crapthings Yes, that's what happened to me, I was also trying the 4x_NMKD-Superscale-SP_178000_G.pth model and got the same error
